### PR TITLE
Code coverage and smell fixes

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

**Review by skoczko — APPROVED** [review]

**Comment by skoczko** [comment]
@adamdickmeiss more code smells though

File: server/src/main/java/org/folio/metastorage/server/OaiService.java [hunk 1]
```diff
@@ -30,12 +30,13 @@ public final class OaiService {
...
   private OaiService() { }
 
-  static final String OAI_HEADER =
-      "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
-          + "<OAI-PMH xmlns=\"http://www.openarchives.org/OAI/2.0/\"\n"
-          + "         xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
-          + "         xsi:schemaLocation=\"http://www.openarchives.org/OAI/2.0/\n"
-          + "         http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd\">\n";
+  static final String OAI_HEADER = """
+  <?xml version="1.0" encoding="UTF-8"?>
```
Comment (skoczko) [inline server/src/main/java/org/folio/metastorage/server/OaiService.java:34]: You could increase the indentation, the minimum indent is stripped.

Comment (adamdickmeiss) [reply server/src/main/java/org/folio/metastorage/server/OaiService.java:34]: I could.. And I would most certainly do it if "auto-indent" for my editor does that. But it doesn't . Visual Studio code?

Comment (adamdickmeiss) [reply server/src/main/java/org/folio/metastorage/server/OaiService.java:34]: #43 .. actually OAI-PMH and xml header should be on same level 0 (without leading spaces)

**Pull request merged**